### ISSUE TITLE
[META] Autopilot systems overhaul

### ISSUE THREAD

**Issue by Benjozork** [flybywiresim/a32nx#2903]
## Execution

It has been suggested that those changes should end up in the `custom-autopilot` branch, which will be offered as experimental then merged into dev. No formal decision yet.

### Flight plan management
- [ ] Import and adaptation of WorkingTitle `FlightPlanManager` implementation (#2831)

### FMGC

Work started on an independent FMGC in #2831

- [ ] Better support for discontinuities `autoflight-fpm`
  - [ ] Implement proper stringing algorithm 
- [ ] Support for storing leg data instead of transforming them into fixes at procedure load time
- [ ] Support for storing WIND data (cc. @viniciusfont)

cc. @MisterChocker
cc. @Lucky38i
cc. @tyler58546

### LNAV / VNAV
 - [ ] LNAV `autopilot-fpm`
   - [ ] Waypoint sequencing
   - [ ] DIRECT TO
   - [ ] Holding patterns
 - [ ] VNAV (RFC: [flybywiresim/rfcs/9](https://github.com/flybywiresim/rfcs/pull/9))

### Autoflight / augmented flight
 - [ ] ~~Custom fly-by-wire in branch `fbw`~~
 - [ ] ~~Custom AP in branch `custom-autopilot`~~
 - [ ] ~~Custom A/THR in branch `athr`~~
   - [ ] ~~Blocked by SimConnect `pContext` bug, reportedly fixed in WU3~~

- FBW/AP/ATHR `autopilot` #3334

**Comment by MisterChocker** [comment]
Id mark this meta issue as outdated since #3334

**Comment by MisterChocker** [comment]
Do we still need this issue?
The FPM and LNAV Guidance is done by #4028 and the individual components are managed by [Autopilot, flight plan manager, managed lateral and vertical navigation](https://github.com/flybywiresim/a32nx/projects/3)

**Comment by derl30n** [comment]
Closing since meta issue does not reflect current development stages.
